### PR TITLE
Initialize roles with seed data

### PR DESCRIPTION
**Pull request by mithmatt**
- Set up initial roles seed data
- Extend EntityRepository to support initSeedData()

### Describe your changes :

See #1940 

#
### Type of change :

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/tags/TagResource.java [hunk 1]
```diff
@@ -93,7 +93,8 @@ public void initialize(CatalogApplicationConfig config) throws IOException {
         tagFile -> {
           try {
             LOG.info("Loading tag definitions from file {}", tagFile);
-            String tagJson = IOUtil.toString(getClass().getClassLoader().getResourceAsStream(tagFile));
+            String tagJson =
+                IOUtil.toString(Objects.requireNonNull(getClass().getClassLoader().getResourceAsStream(tagFile)));
             TagCategory tagCategory = JsonUtils.readValue(tagJson, TagCategory.class);
```
Comment (mithmatt) [inline catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/tags/TagResource.java:98]: TagRepository does not inherit from EntityRepository and the entity Tag is different from TagCategory.

Hence we cannot easily use the new initSeedData API in EntityRepository

File: bootstrap/sql/mysql/v003__create_db_connection_info.sql [hunk 2]
```diff
@@ -290,3 +290,5 @@ DROP COLUMN deactivated,
 ADD COLUMN deleted BOOLEAN GENERATED ALWAYS AS (JSON_EXTRACT(json, '$.deleted')),
 ADD INDEX (deleted);
 
+UPDATE policy_entity
```
Comment (mithmatt) [inline bootstrap/sql/mysql/v003__create_db_connection_info.sql:293]: @sureshms  I'm pretty new to flyway
Making these changes based on my limited understanding.

**Comment by akash-jain-10** [comment]
@mithmatt  - Can you please update the version for `openmetadata-ingestion-core` package to below code in _version.py file located at `ingestion-core/src/metadata/_version.py`.
Ref - https://github.com/mithmatt/OpenMetadata/blob/aa6485533135714792dc9c55731a2052d1814717/ingestion-core/src/metadata/_version.py#L10
From: 
```
Version("metadata", 0, 8, 0, dev=0)
```

To:

```
Version("metadata", 0, 8, 0, dev=1)
```

**Comment by mithmatt** [comment]
> @mithmatt - Can you please update the version for `openmetadata-ingestion-core` package to below code in _version.py file located at `ingestion-core/src/metadata/_version.py`. Ref - https://github.com/mithmatt/OpenMetadata/blob/aa6485533135714792dc9c55731a2052d1814717/ingestion-core/src/metadata/_version.py#L10 From:
> 
> ```
> Version("metadata", 0, 8, 0, dev=0)
> ```
> 
> To:
> 
> ```
> Version("metadata", 0, 8, 0, dev=1)
> ```

I can , but how is it related to this change?
Is it because of schema change? @akash-jain-10

**Comment by mithmatt** [comment]
Just came to know about #1875 (Wasn't paying attention to my github updates, will read through and try to figure out what is going on)

**Comment by akash-jain-10** [comment]
> Details

This is related to [CI](https://github.com/open-metadata/OpenMetadata/runs/4836382222?check_suite_focus=true) which is failing to upload dev version of `openmetadata-ingestion-core` to PyPi. It seems version `0.8.0.dev1` already exists and CI tries to bump up the same version for your branch to `0.8.0.dev1`.

**Comment by mithmatt** [comment]
My latest attempt to fix the v003 sql failed the migration

```
2022-01-17T06:03:33.8441013Z Migration v003__create_db_connection_info.sql failed
2022-01-17T06:03:33.8441419Z ----------------------------------------------------
2022-01-17T06:03:33.8441669Z SQL State  : HY000
2022-01-17T06:03:33.8441887Z Error Code : 3105
2022-01-17T06:03:33.8442565Z Message    : The value specified for generated column 'fullyQualifiedName' in table 'policy_entity' is not allowed.
2022-01-17T06:03:33.8443433Z Location   : db/migration/v003__create_db_connection_info.sql (/home/runner/work/OpenMetadata/OpenMetadata/catalog-rest-service/target/test-classes/db/migration/v003__create_db_connection_info.sql)
2022-01-17T06:03:33.8444191Z Line       : 293
2022-01-17T06:03:33.8444441Z Statement  : UPDATE policy_entity
2022-01-17T06:03:33.8543508Z SET fullyQualifiedName = JSON_EXTRACT(json, '$.name')
2022-01-17T06:03:33.8543959Z 
2022-01-17T06:03:33.8544601Z Caused by: java.sql.SQLException: The value specified for generated column 'fullyQualifiedName' in table 'policy_entity' is not allowed.
2022-01-17T06:03:33.8544913Z 
```

**Comment by mithmatt** [comment]
Test failure `Publish openmetadata-ingestion-core packages / build-and-push (pull_request) Failing after 54s — build-and-push` is being fixed separately ...

**Pull request merged**